x-pack/plugins/security_solution/package.json

```diff
@@ -11,7 +11,7 @@
     "cypress:open": "TZ=UTC node ./scripts/start_cypress_parallel open --spec './cypress/e2e/**/*.cy.ts' --config-file ./cypress/cypress.config.ts --ftr-config-file ../../../../../../x-pack/test/security_solution_cypress/cli_config",
     "cypress:open:ccs": "yarn cypress:open --config specPattern=./cypress/ccs_e2e/**/*.cy.ts",
     "cypress:open:upgrade": "yarn cypress:open --config specPattern=./cypress/upgrade_e2e/**/*.cy.ts",
-    "cypress:run": "yarn cypress:run:reporter --browser chrome --spec './cypress/e2e/**/*.cy.ts'; status=$?; yarn junit:merge && exit $status",
+    "cypress:run": "yarn cypress:run:reporter --browser chrome --spec './cypress/e2e/!(investigations|explore)/**/*.cy.ts'; status=$?; yarn junit:merge && exit $status",
```

Comment:
The main `cypress:run` command no longer executes investigations and explore tests. I think we should consider moving this logic to the buildkite files. It's confusing for a generically named command not to run all the tests.